tests/functional/MigrateRunnerTest.php

```diff
@@ -0,0 +1,406 @@
+<?php
+
+namespace Unish;
+
+use Webmozart\PathUtil\Path;
+
+/**
+ * @group slow
+ * @group commands
+ * @coversDefaultClass \Drush\Drupal\Commands\core\MigrateRunnerCommands
+ */
+class MigrateRunnerTest extends CommandUnishTestCase
```

Comment:
Any chance this can extend UnishIntegrationTestCase? See \Unish\AnnotatedCommandCase as an example that also brings in woot module. Integration tests dont reinstall drupal so they are faster.

Reply:
Well, right now the conversion is not straight as many some tests require clean start but I see your point. As this PR is already big, is it a possibility that we're doing this in a followup?

Reply:
Yes, thats fine